docs/bundler/migration.md

```diff
@@ -0,0 +1,1107 @@
+
+```bash
+$ esbuild <entrypoint> --outdir=out --bundle
+$ bun build <entrypoint> --outdir=out
+```
+
+In Bun's CLI, simple boolean flags like `--minify` do not accept an argument. Other flags like `--outdir <path>` do accept an argument; these flags can be written as `--outdir out` or `--outdir=out`. Some flags like `--define` can be specified several times: `--define foo=bar --define bar=baz`.
+
+{% table %}
+
+- `esbuild`
+- `bun build`
+
+---
+
+- `--bundle`
+- n/a
+- Not necessary, `bun build` always bundles.
+
+---
+
+- `--define:K=V`
+- `--define K=V`
+- Small syntax difference; no colon.
+
+  ```bash
+  $ esbuild --define:K=V
+  $ bun build --define K=V
+  ```
+
+---
+
+- `--external:<pkg>`
+- `--external <pkg>`
+- Small syntax difference; no colon.
+
+---
+
+- `--format`
+- `--format`
+- Bun only supports `"esm"` currently but other module formats are planned. esbuild defaults to `"iife"`.
+
+---
+
+- `--loader`
+- `--loader`
+- Small syntax difference.
+
+  ```bash
+  $ esbuild app.ts --bundle --loader:.svg=text
+  $ bun build app.ts --loader .svg:text
+  ```
+
+---
+
+- `--minify`
+- `--minify`
+- No differences
+
+---
+
+- `--outdir`
+- `--outdir`
+- No differences
+
+---
+
+- `--outfile`
+- `--outfile`
+
+---
+
+- `--packages`
+- n/a
+- Not supported
+
+---
+
+- `--platform`
+- `--target`
+- Renamed to `--target` for consistency with tsconfig
+
+---
+
+- `--serve`
+- n/a
+- Not applicable.
+
+---
+
+- `--sourcemap`
+- `--sourcemap`
+- No differences
+
+---
+
+- `--splitting`
+- `--splitting`
+- No differences
+
+---
+
+- `--target`
+- n/a
+- No supported. Bun's bundler performs no syntactic downleveling at this time.
+
+---
+
+- `--watch`
+- n/a
+- Not applicable.
+
+---
+
+- `--allow-overwrite`
+- n/a
+- Overwriting is always allowed
+
+---
+
+- `--analyze`
+- n/a
+- Not supported. Use `--manifest` to generate a manifest file.
+
+---
+
+- `--asset-names`
+- `--asset-naming`
+- Renamed for consistency with `naming` in JS API
+
+---
+
+- `--banner`
+- n/a
+- Not supported
+
+---
+
+- `--certfile`
+- n/a
+- Not applicable, Bun's bundler does
+
+---
+
+- `--charset=utf8`
+- n/a
+- Not supported
+
+---
+
+- `--chunk-names`
+- `--chunk-naming`
+- Renamed for consistency with `naming` in JS API
+
+---
+
+- `--color`
+- n/a
+- Always enabled
+
+---
+
+- `--drop`
+- n/a
+- Not supported
+
+---
+
+- `--entry-names`
+- `--entry-naming`
+- Renamed for consistency with `naming` in JS API
+
+---
+
+- `--footer`
+- n/a
+- Not supported
+
+---
+
+- `--global-name`
+- n/a
+- Not applicable, Bun does not support `iife` output at this time.
+
+---
+
+- `--ignore-annotations`
+- n/a
+- Not supported
+
+---
+
+- `--inject`
+- n/a
+- Not supported
+
+---
+
+- `--jsx`
+- `--jsx-runtime`
+- Supports `"automatic"` (uses `jsx` transform) and `"classic"` (uses `React.createElement`)
+
+---
+
+- `--jsx-dev`
+- n/a
+- Bun uses `jsxDEV` by default. Set `NODE_ENV=production` to use the production `jsx` transform instead.
```

Comment:
oh wait i forgot that in addition to this, bun (and i think esbuild) read the tsconfig. not sure how to convey that here without bloating up the sentence.